### PR TITLE
Adding 1.24 bug triage shadows to team list

### PR DESCRIPTION
**Pull request by jyotimahapatra**
Adding @DiptoChakrabarty @helayoty @doshyt @ritpanjw to 1.24 Bug Triage Shadow list.

### CODE REVIEW

File: releases/release-1.24/release-team.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
...
 | Enhancements | Grace Nguyen ([@gracenng](https://github.com/gracenng) / Slack: `@Grace Nguyen`) | |
 | CI Signal | Leonard Vincent Simon Pahlke ([@leonardpahlke](https://github.com/leonardpahlke) / Slack: `@lvspahlke`) | |
-| Bug Triage | Jyoti Mahapatra ([@jyotimahapatra](https://github.com/jyotimahapatra) / Slack: `@Jyoti Mahapatra`) | |
+| Bug Triage | Jyoti Mahapatra ([@jyotimahapatra](https://github.com/jyotimahapatra) / Slack: `@Jyoti Mahapatra`) | Dipto Chakrabarty ([@DiptoChakrabarty (https://github.com/DiptoChakrabarty)] / Slack: `@Dipto`) | Heba Elayoty ([@helayoty](https://github.com/helayoty) / Slack: `@helayoty`) | Igor Andriushchenko ([@doshyt](https://github.com/doshyt) / Slack: `@Igor Andriushchenko`) | Ritu Panjwani ([@ritpanjw](https://github.com/ritpanjw) / Slack: `@ritpanjw`) |
```
Comment (leonardpahlke) [inline releases/release-1.24/release-team.md:8]: nit
There are too many pipe symbols (`|`) replace the pipe between shadows with a comma (`,`) - see section Docs below as reference

**Review by leonardpahlke — CHANGES_REQUESTED** [review]

File: releases/release-1.24/release-team.md (hunk 1 above)
Comment (jyotimahapatra) [reply releases/release-1.24/release-team.md]: ah, didnt notice that. Thanks!

**Pull request merged**